### PR TITLE
Spaces for blocks not hashes

### PR DESCRIPTION
**Pull request by Dfenniak**

### CODE REVIEW

**Comment by Dfenniak** [comment]
PTAL

File: README.md [hunk 1]
```diff
@@ -244,18 +244,24 @@ Translations of the guide are available in the following languages:
...
-  the advantage of adding visual difference between block and hash
-  literals. Whichever one you pick - apply it consistently.
+  For hash literals never leave a space after `{` and before `}`.
+  This resuts in a simple visual indicator of what's a hash vs. a block.
 
   ```Ruby
-  # good - space after { and before }
-  { one: 1, two: 2 }
+  # bad
+  { :one => 1, :two => 2}
```
Comment (andy-j) [inline README.md]: `{ :one => 1, :two => 2 }` ?

**Comment by RawStewage** [comment]
This is a good idea. I think that anything to disambiguate the over loaded `{}` symbols is a good thing.

**Comment by benzittlau** [comment]
:+1:

**Comment by andy-j** [comment]
👍

**Pull request merged**